src/main/java/tamaized/voidscape/Voidscape.java

```diff
@@ -360,5 +372,5 @@ private static EntityHitResult getEntityHitResult(Entity shooter, Vec3 startVec,
 
 		return entity == null ? null : new EntityHitResult(entity, vector3d);
 	}
-
+	
```

Comment:
Weird spacing was added here

Reply:
There shouldn't be any diff here at all

Reply:
I just added a new line and since it was at the brace (line 362/374), it got indented. Kind of annoying lol.